guidelines/goals.md

```diff
@@ -0,0 +1,71 @@
+# Goals and Non-Goals
+
+This document defines the purpose of the Message Format Working Group (MFWG)
+and informs the decisions about the scope and the priorities of its efforts.
+
+## Goals
+
+The primary task of the MFWG is to develop an industry standard for the
+representation of localizable message strings. The design goals are listed
+below.
+
+ 1. Express complex grammatical features, such as plurals, genders, and
+    inflections, catalogued in the CLDR.
+
+ 2. Express other variance in translation, due to other linguistic and regional
+    features not listed in the CLDR, as well as due to the presentation media,
+    context, circumstance, and other factors.
+
+ 3. Represent structured data alongside translations, like markup, comments,
+    and metadata.
+
+ 4. Ensure interoperability with existing interchange formats, in particular
+    with XLIFF.
+
+ 5. Ensure that machine translation and translation memory solutions can be
+    used for authoring translations.
+
+ 6. Enable a wide range of user space solutions building on top of the
+    standard, manifesting different ideas and programming paradigms, and
+    optimized for different uses and audiences.
+
+
+## Deliverables
+
+ 1. A formal definition of the canonical data model for representing
+    translations.
```

Comment:
Did you mean "messages" instead of "translations", here?  I think the data model describes the inputs to the message formatting API <-> the localizable message strings of interest.  ("Translation" makes me think of what the translators provide on the l10n side per message / message variant per target locale.)

If so, then the same comment applies above and below throughout.

Reply:
Ah, good point. I wrote `translations` because I tend to think that developers who create messages are the first translators :) I'll use _messages_ here.

At this point, I wouldn't want to make a decision about whether the data model is just for message units, or perhaps for collections of units (resources) too. Do you think that the following phrasing is generic enough?

> A formal definition of the canonical data model for representing messages.

Reply:
I went with the following, but I'm open to suggestions about how to make this better.

> A formal definition of the canonical data model for representing localizable message strings.

Reply:
Yep, that sounds good to me.